src/main/java/org/alfresco/rest/api/impl/QuickShareLinksImpl.java

```diff
@@ -618,6 +655,22 @@ private QuickShareLink getQuickShareInfo(NodeRef nodeRef, Map<String, Object> ma
         }
     }
 
+    private Map<String, Object> filterProps(Map<String, Object> properties, List<QName> toRemove)
+    {
+        Map<String, Object> filteredProps = new HashMap<>(properties);
+        List<String> propsToRemove = toRemove.stream().map(e -> e.toPrefixString(namespaceService)).collect(Collectors.toList());
+        filteredProps.keySet().removeAll(propsToRemove);
+        return filteredProps;
+    }
+
+    private Node getNode(NodeRef nodeRef, List<String> includeParam, Map<String, UserInfo> mapUserInfo)
+    {
+        List<String> modifiedIncludeParams = new LinkedList<>(includeParam);
+        // Remove path as this information is already retrieved elsewhere
+        modifiedIncludeParams.remove(PARAM_INCLUDE_PATH);
+        return nodes.getFolderOrDocument(nodeRef, null, null, includeParam, mapUserInfo);
```

Comment:
This was supposed to be modifiedIncludeParams instead of includeParam

Reply:
Good spot! Thanks